static/js/settings_muted_users.js

```diff
@@ -0,0 +1,24 @@
+import $ from "jquery";
+
+import * as muting_ui from "./muting_ui";
+
+export let loaded = false;
+
+export function set_up() {
+    loaded = true;
+    $("body").on("click", ".settings-unmute-user", function (e) {
+        const $row = $(this).closest("tr");
+        const user_id = Number.parseInt($row.attr("data-user-id"), 10);
+
+        e.stopImmediatePropagation();
+
+        muting_ui.unmute_user(user_id);
+        $row.remove();
```

Comment:
I don't think we should do this UI update here of removing the row -- let's just let the live-update handle it.

Structurally, we have to make live-update work because you can have multiple browser tabs open and looking at this screen (Worth testing that BTW), and so generally it's better for buttons like this to only send the request to the server (and maybe do a loading spinner, if useful), and then have the success handler for that HTTP request just hide the loading spinner.  

Anything in the existing codebase that doesn't have that structure and isn't local echo for message feed is legacy code that we've neglected to update.